src/testing.rst

```diff
@@ -51,9 +174,10 @@ the root repository directory and run, the ``scripts/run_docker_build.sh`` scrip
     $ ./scripts/run_docker_build.sh
 
 
-
 Should a Recipe Run All of a Package's Tests?
 ---------------------------------------------
+
 No. A recipe does not have to run all of a package's unit or integration tests.
 Sometimes, this might even be impossible due to timeouts on CI services.
 Just test sufficiently to know that the package is in good working order.
+
```

Comment:
nit: Maybe this is just due to Windows formatting, in which case don't worry about it. Do we need this extra newline?